Yotpo/Yotpo/composer.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "yotpo/module-yotpo",
```

Comment:
@omryYotpo does the name is the name of the module in the file system? or can we put a proper name?
and it seems that we can do a composer package to this and distribute this over compose.io